components/search/MoreFacetsDropdownOptions.vue

```diff
@@ -93,6 +114,15 @@
     methods: {
       selectedHandler(value) {
         this.$emit('selectedOptions', this.name, value);
+      },
+
+      // Takes hex value `#000000` and converts it to it's Standard Color Name `Black`
+      colorHexToStandardColorName(label) {
```

Comment:
There will be more facets in this dropdown requiring localisation of labels, so this should be made more generic:
```diff
diff --git a/components/search/MoreFacetsDropdownOptions.vue b/components/search/MoreFacetsDropdownOptions.vue
index fc59ae8..1097720 100644
--- a/components/search/MoreFacetsDropdownOptions.vue
+++ b/components/search/MoreFacetsDropdownOptions.vue
@@ -26,7 +26,7 @@
           :data-qa="`colour swatch ${colorHexToStandardColorName(filter.label)}`"
           :aria-label="`${$t('colourSwatch')} - ${colorHexToStandardColorName(filter.label)}`"
         />
-        {{ colorHexToStandardColorName(filter.label) }}
+        {{ localiseFilterLabel(filter.label) }}
         <span
           class="reset icon-close"
           :aria-label="$t('facets.button.reset')"
@@ -50,7 +50,7 @@
             :data-qa="`colour swatch ${colorHexToStandardColorName(filter.label)}`"
             :aria-label="`${$t('colourSwatch')} - ${colorHexToStandardColorName(filter.label)}`"
           />
-          {{ colorHexToStandardColorName(filter.label) }}
+          {{ localiseFilterLabel(filter.label) }}
           <span
             class="reset icon-close"
             :aria-label="$t('facets.button.reset')"
@@ -112,6 +112,11 @@
     },
 
     methods: {
+      localiseFilterLabel(label) {
+        const key = `facets.${this.name}.options.${label}`;
+        return this.$te(key) ? this.$t(key) : label;
+      },
+
       selectedHandler(value) {
         this.$emit('selectedOptions', this.name, value);
       },
```
Then get rid of `colorHexToStandardColorName`, and update the keys in lang/en.js to include the `#`.